pattoo/api/portal/purge.py

```diff
@@ -0,0 +1,102 @@
+"""Routes for purge."""
+
+# Flask imports
+from flask import Blueprint
+from flask import request, session
+
+# Standard import
+from sqlalchemy import func
+import datetime
+import time
+
+# Pattoo imports
+from pattoo.db import db
+from pattoo.db.models import Data as DataModel
+
+# Import server resources
+from .forms import PurgeForm
+
+# Define the AGENTS global variable
+PURGE = Blueprint('PURGE', __name__)
+
+@PURGE.route('/api/purge', methods=['GET', 'POST'])
+def datapoints():
+    """
+    Handle purging of data.
+
+    Args:
+        None
+
+    Returns:
+        response (dict): Response Message
+
+    """
+    # Check if a user was stored in session first
+    if session.get('idx_user', None) is None:
+        response = {'data': {'message': 'Login first'}}
+        # Block access if no user was found in session
+        return response
+
+    response = {'data':{'message': 'Query did not run'}}
```

Comment:
This should be initialized at the top. It's the default unsuccessful response. All the other responses should have success correct?

Setting responses throughout the code, without immediately returning the value makes the code hard to follow and ultimately troubleshoot.